python/geo2png.py

```diff
@@ -0,0 +1,121 @@
+# trafo -> png
+# Original source https://github.com/vrandezo/wikidata-analytics/blob/6896d639bd124eec7571f1179f27ae2b6bfaa435/geo2png.py
```

Comment:
Yes, I confirm, my original code was published into the Public Domain, or, if that leaves any legal doubt, under CC0.